### DISCUSSION TITLE
Cannot login with Github

### DISCUSSION THREAD

**Discussion by jacky-lam** [vercel/community#1737] [Help] [answered]
### Summary

Since the issue with Vercel this morning - I am not longer able to log into Vercel with my GitHub account
- I get this error message when I try to log in via Github (screenshot below)
- I have tried clearing my cache, session storage, local storage, cookies - this still did not work
- I have tried logging in from an incognito window - still get the same error after authenticating the github account

URL: https://vercel.com/login

<img width="897" alt="Screenshot 2023-03-09 at 17 16 07" src="https://user-images.githubusercontent.com/17092881/224105645-7a6ccfc2-2a0a-4ede-afdc-f5a42bc8af45.png">

**Comment by hellojennifertran** [comment, marked as answer]
Hello! Can you try logging in via email and see if that works for you? If not, please open a support case: https://vercel.com/help

**Reply by jacky-lam** [reply]
Thank you, this did work.

It seems to still be broken to login via Github, but the email way does work!
